frontend/packages/knative-plugin/src/components/services/ServiceDetailsPage.tsx

```diff
@@ -46,7 +46,7 @@ const ServiceDetails: React.FC<{ obj: ServiceKind }> = ({ obj }) => {
   return (
     <>
       <div className="co-m-pane__body">
-        <SectionHeading text={t('knative-plugin~Service details')} />
+        <SectionHeading text={t('knative-plugin~Details')} />
```

Comment:
We usually use `$Kind details`. Please keep "Service details", or we can show "Functions details if needed". 

We should check this as well with Naina and Ethan.

Reply:
This is as requested by Ethan.

https://redhat-internal.slack.com/archives/C05MDC1T35J/p1697201614975339?thread_ts=1697132910.238989&cid=C05MDC1T35J